plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js

```diff
@@ -279,9 +296,9 @@ Ext.define('NX.coreui.controller.Capabilities', {
 
     if (list) {
       me.getCapabilityTypeStore().load(
-        function() {
-          me.reselect();
-        }
+          function() {
```

Comment:
Tnx for the followup

On Thu, Feb 5, 2015 at 10:02 AM, Alin Dreghiciu <notifications@github.com>
wrote:

> In
> plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js
> <https://github.com/sonatype/nexus-oss/pull/1024#discussion_r24183668>:
>
> > @@ -279,9 +296,9 @@ Ext.define('NX.coreui.controller.Capabilities', {
> >
> >      if (list) {
> >        me.getCapabilityTypeStore().load(
> > -        function() {
> > -          me.reselect();
> > -        }
> > +          function() {
>
> Found it: http://screencast.com/t/2uWRgBkl
> And that is in out standard code style
> https://github.com/sonatype/codestyle/blob/master/sonatype-idea.xml
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sonatype/nexus-oss/pull/1024/files#r24183668>.
>